plugins/dashboard/views/status.html

```diff
@@ -0,0 +1,116 @@
+<html>
+<title>Dashboard</title>
```

Comment:
\<head>\</head> ?

Reply:
It's the new thing ... headless web... was that not what that meant? :-) Added missing head :-)